lib/rack/timeout/core.rb

```diff
@@ -53,8 +53,6 @@ def read_timeout_property value, default
       when nil   ; read_timeout_property default, default
       when false ; false
       when 0     ; false
-      when String
-        read_timeout_property value.to_i, default
```

Comment:
This functionality was released in 0.6.0 (and we made changes in our app thinking this would stick around in rack-timeout). I've added it back in https://github.com/sharpstone/rack-timeout/pull/165

Reply:
At least I tried 🤷‍♀️ :)

Reply:
I left another comment on your PR. I could be open to revisiting accepting strings into the config, but we need to take more care to handle some of the special cases that I wasn't handling in my original change to the config. Changing the way we're doing config should be at least a minor bump (instead of patch).

Reply:
I'm not using rack-timeout anymore so I'm unlikely to continue with the PR